test/api/detection_test.cpp

```diff
@@ -426,178 +446,182 @@ TEST(junction_detection, analyze_sa_tag)
                        1000,                    // default partition_max_distance
                        0.5};                    // default hierarchical_clustering_cutoff
 
-    std::vector<Junction> junctions_res{};
+    { // Example 1
+        seqan3::debug_stream << "----------------------------------First Example:----------------------------------\n";
```

Comment:
This is printed on each test execution – is this intended (even if `print_compare_junction_vectors` is just a comment)? The same below for the second example.